src/pages/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -6,13 +8,13 @@ import getDay from 'date-fns/getDay';
 import * as enUS from 'date-fns/locale';
 import parse from 'date-fns/parse';
 import startOfWeek from 'date-fns/startOfWeek';
-import { MutableRefObject, useCallback, useMemo, useRef, useState } from 'react';
 import 'react-big-calendar/lib/sass/styles.scss';
-import '../../shared/styles/CalendarStyles.scss';
 import { Calendar, dateFnsLocalizer, Event, EventProps, ToolbarProps } from 'react-big-calendar';
 import { useParams } from 'react-router';
 import { RRule } from 'rrule';
 
+import 'lib/styles/CalendarStyles.scss';
```

Comment:
Could this file be put into a `styles` directory for just the calendar component?